shared/templates/template_BASH_kernel_module_disabled

```diff
@@ -4,4 +4,9 @@
 # complexity = low
 # disruption = medium
 # THIS FILE IS GENERATED. DO NOT EDIT!
```

Comment:
I am really confused now. The comment says you shouldn't edit the file directly. Yet this PR edits the file directly and doesn't edit any template or other file. So it seems it's done contrary to the warning in the generated file. Am I missing something?